From 9bcc2410607a60b9716bffbfc0093bfab3037a52 Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Thu, 7 May 2026 17:45:32 +0800
Subject: [PATCH] configs: evb-ast2705: run bootmac for MAC boot

Extend the default boot command to run bootmac when boot_device is mac.
This wires the AST2705 MAC recovery selection into the EVB default
boot flow.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Ia50da7568874e4ef998da5b45c7a5ef330c3bada
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 configs/evb-ast2705_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/evb-ast2705_defconfig b/configs/evb-ast2705_defconfig
index a1ada5a91d..1954e04b12 100644
--- a/configs/evb-ast2705_defconfig
+++ b/configs/evb-ast2705_defconfig
@@ -37,7 +37,7 @@ CONFIG_LEGACY_IMAGE_FORMAT=y
 CONFIG_USE_BOOTARGS=y
 CONFIG_BOOTARGS="console=ttyS12,115200n8 root=/dev/ram rw earlycon"
 CONFIG_USE_BOOTCOMMAND=y
-CONFIG_BOOTCOMMAND="echo Boot from ${boot_device}; if test ${boot_device} = mmc; then run bootmmc; fi; if test ${boot_device} = spi; then run bootspi; fi; if test ${boot_device} = ufs; then run bootufs; fi;"
+CONFIG_BOOTCOMMAND="echo Boot from ${boot_device}; if test ${boot_device} = mmc; then run bootmmc; fi; if test ${boot_device} = spi; then run bootspi; fi; if test ${boot_device} = ufs; then run bootufs; fi; if test ${boot_device} = mac; then run bootmac; fi;"
 CONFIG_DISPLAY_BOARDINFO_LATE=y
 CONFIG_HUSH_PARSER=y
 # CONFIG_AUTO_COMPLETE is not set
-- 
2.43.0

